index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="src/style.css">
+    <title>About RealCoolKids</title>
+</head>
+<body>
+    <h1>About RealCoolKids</h1>
+    <p>These are the RealCoolKids, and this is a list of their favorite things.</p>
+    <ul>
+        <li><a href="#Jack">Jack</a></li>
+        <li><a href="#Denise">Denise</a></li>
+        <li><a href="#Erin">Erin</a></li>
+        <li><a href="#Leigh-Ann">Leigh-Ann</a></li>
+    </ul>
+    <hr>
+    <div id="Jack">
+        <h2>Jack</h2> 
+        <img src="assets/skiing.jpg" alt="waterskiing">
```

Comment:
consider making your alt more descriptive - a screen reader will use this to describe the image to visually impaired individuals.